xen-platform: Add device_id for Windows VMs

The platform device has a different device id for Windows VMs so match
this one too.
diff --git a/OvmfPkg/XenIoPciDxe/XenIoPciDxe.c b/OvmfPkg/XenIoPciDxe/XenIoPciDxe.c
index 15803eb847..e764ece34f 100644
--- a/OvmfPkg/XenIoPciDxe/XenIoPciDxe.c
+++ b/OvmfPkg/XenIoPciDxe/XenIoPciDxe.c
@@ -30,6 +30,7 @@
 
 #define PCI_VENDOR_ID_XEN                0x5853
 #define PCI_DEVICE_ID_XEN_PLATFORM       0x0001
+#define PCI_DEVICE_ID_XEN_PLATFORM_XS    0x0002
 
 /**
 
@@ -102,7 +103,8 @@ XenIoPciDeviceBindingSupported (
 
   if (Status == EFI_SUCCESS) {
     if ((Pci.Hdr.VendorId == PCI_VENDOR_ID_XEN) &&
-        (Pci.Hdr.DeviceId == PCI_DEVICE_ID_XEN_PLATFORM)) {
+        (Pci.Hdr.DeviceId == PCI_DEVICE_ID_XEN_PLATFORM ||
+         Pci.Hdr.DeviceId == PCI_DEVICE_ID_XEN_PLATFORM_XS)) {
       Status = EFI_SUCCESS;
     } else {
       Status = EFI_UNSUPPORTED;
